mifosng-android/src/main/res/layout/fragment_add_loan.xml

```diff
@@ -179,18 +166,31 @@
                 android:spinnerMode="dropdown"
                 android:visibility="gone" />
 
-            <android.support.design.widget.TextInputLayout style="@style/TextInput.Base">
-
-                <EditText
-                    android:id="@+id/et_nominal_interest_rate"
-                    style="@style/EditText.BaseWidth"
-                    android:gravity="start"
-                    android:hint="@string/nominal_interest_rate"
-                    android:imeOptions="actionNext"
-                    android:inputType="number"
-                    android:paddingBottom="16dp"
-                    android:singleLine="true" />
-            </android.support.design.widget.TextInputLayout>
+            <LinearLayout style="@style/LinearLayout.Width">
+
+                <android.support.design.widget.TextInputLayout
+                    style="@style/TextInput.Base.Weight"
+                    android:layout_weight="0.3">
+
+                    <EditText
+                        android:id="@+id/et_nominal_interest_rate"
+                        android:layout_height="wrap_content"
+                        android:layout_width="match_parent"
+                        android:gravity="start"
+                        android:hint="@string/nominal_interest_rate"
+                        android:imeOptions="actionNext"
+                        android:inputType="number"
```

Comment:
Would this also support decimals out of the box ? or do you have to supply the flag `TYPE_NUMBER_FLAG_DECIMAL`